src/components/Navigation.js

```diff
@@ -0,0 +1,20 @@
+import "./Navigation.css";
+import { NavLink } from "react-router-dom";
+
+export default function Navigation() {
+  return (
+    <section>
+      <nav className="Nav">
+        <NavLink className="Navlink" to="/profile">
+          profile
+        </NavLink>
+        <NavLink className="Navlink" to="/places">
+          places
+        </NavLink>
+        <NavLink className="Navlink" to="/map">
+          map
+        </NavLink>
+      </nav>
+    </section>
```

Comment:
im not sure if section is needed here... since you are already wrapping the navlinks inside the nav.

Reply:
noted! thanks 🙃